core/src/main/java/org/elasticsearch/node/InternalSettingsPreparer.java

```diff
@@ -44,6 +47,9 @@
 
 public class InternalSettingsPreparer {
 
+    private static Logger logger = ESLoggerFactory.getLogger("settings");
```

Comment:
There are a few legitimate uses for this method, namely the loggers that we provide to Netty, and the naming for the circuit breaker loggers, as well as setting logging levels via the API (all we get there from the user input is the name of the logger, so we have to have a way to look up loggers by name).